dataingest/spatial/import.sh

```diff
@@ -2,21 +2,14 @@
 
 set -e
 
-# Docs say this shouldn't work on Mac Docker Desktop.
-# https://docs.docker.com/network/host/
-# But it works for me on Docker Desktop 4.4.2.
-#EXTRA=--network="host"
-
-DOCKER="osgeo/gdal:alpine-small-3.3.3"
-
 declare -A tables
```

Comment:
this doesn't work in zsh: https://stackoverflow.com/questions/30269722/declaring-arrays-in-zsh
trying to see if theres an alternative that does...